src/plays/youtube-clone/youtube.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+const KEY = 'AIzaSyCO5HO9G8OyvzdnQn8DGSkfwzQIrXkYo0s'
```

Comment:
Do not expose the keys. It may bring security issues to your account.

Please create a file called `.env` at the root of the project folder and keep the key there instead. You can read more about it from here: https://create-react-app.dev/docs/adding-custom-environment-variables/